test/unit/chunkedstream_spec.js

```diff
@@ -0,0 +1,806 @@
+/* globals expect, it, describe, beforeEach, Stream, ChunkedStreamContinuous,
+   ChunkedStreamFragmented, Dict,  MissingDataException,
+   ALLOCATE_NO_CHUNKS_SIZE, ChunkedStreamBase, jasmine */
+
+'use strict';
+
+describe('stream', function () {
+  var arrayComparerThatFixesJasmineIssue786 = function (first, second) {
+
+    var isFirstArray = Array.isArray(first) ||
+      Object.prototype.toString.call(first.buffer) === '[object ArrayBuffer]';
+    var isSecondArray = Array.isArray(second) ||
+      Object.prototype.toString.call(second.buffer) === '[object ArrayBuffer]';
+
+    if (isFirstArray || isSecondArray) {
+
+      var i = first.length;
+      if (i !== second.length) {
+        return false;
+      }
+
+      while (i--) {
+        if (first[i] !== second[i]) {
+          return false;
+        }
+      }
+      return true;
+    }
+  };
+
+  beforeEach(function() {
```

Comment:
Is it necessary to register this before every test, couldn't you do it once instead? E.g.
```javascript
beforeAll(function (done) {
  jasmine.addCustomEqualityTester(arrayComparerThatFixesJasmineIssue786);
  done();
});
```